### ISSUE TITLE
How to best create and present community-originated content vs team produced content

### ISSUE THREAD

**Issue by aredehel** [vegaprotocol/community#18]
Our current content channels are 
- Discord 
- Forum 
- Community Update 
- Twitter 
- Medium 

To recap purpose and audience see here: https://www.notion.so/vegaprotocol/e69cfb9b66374638a871fb78c403d514?v=36b31334c5b74c44bea98dbecebfa09d

I propose that we add either one or two of the following channels. 
1. Community Portal/Learn/Resource

A place for community-created created content aimed at providing educational resources related to Vega. The types of content I'm imagining is the likes of ELI5, walk-throughs, summaries and links to the twitch events, outline of community core and how to get involved ++ Whilst this type of content could go on the forum the Discourse format is not particularly well-suited to host multi media content, linking and being 'searched'.

Examples of projects that have a community page somewhat like this:

My favourite version of this is makers community portal, entirely created by their community core team and covers a wide array of content about Maker. 

- [https://community-development.makerdao.com/en/]

another way that is less about sharing content but more as a hub for directions is something like Near's github. 

- [https://github.com/near/community](https://github.com/near/community)

somewhat similar to Near is metacartel github. I don't love the structure but still a way of hosting community content. 

- [https://github.com/metacartel/mission/wiki](https://github.com/metacartel/mission/wiki)

I envision the audience for the community hub/portal/name tbc to be primarily crypto native, DeFi familiar potential community members interested in learning more about Vega and the community.

2. Community section in Medium. 

See Marcin's input in the Discord channel.

**Comment by candida-d** [comment]
A new community-driven portal sounds like a great idea. 

I think the Maker portal is SUPER sleek! But one thing stands out - you can't just jump in and contribute, you need to go through the process of introducing yourself, etc. 

And that's not bad, but I think that comes down to what the goals are for how community contributes (outside of community core) and whether or not we want a more open style of contribution (raise an issue/pull request), or want to get to know people before they contribute (send us a message, introduce yourself, pitch your idea). 

I like the idea of a GitHub-focused area but the risk is it could put off people who want a sleeker/more exciting look - or think it's too techy for them. 

Medium requires the least amount of up-front work (bonus), and is easy to link to/share, but I think as a format it is a bit limiting, there's not a lot of room for anything other than blog-style content.

**Comment by candida-d** [comment]
One other thing to consider is how to highlight other types of content like a community created docs-type wiki (example: https://dyor-crypto.fandom.com/wiki/Maker_DAO), and I could see how a section of the main site (like Maker has) can link off to other resources like that.

**Comment by aredehel** [comment]
Decision (08/02) let's use what we have until a need arises for something else. We'll expand on the readme on the front page to include an overview of community, other links and perhaps add the glossary.
